### DISCUSSION TITLE
how to solve this?

### DISCUSSION THREAD

**Discussion by ravenkim** [typescript-eslint/typescript-eslint#9682] [Community Feedback]
`
p6fe30 → ✓ @typescript-eslint/type-utils@npm:7.17.0 [61552] doesn't provide @types/supports-color to debug@npm:4.3.5 [1ff4b]
p5042e → ✓ @typescript-eslint/type-utils@npm:7.17.0 [61552] doesn't provide supports-color to debug@npm:4.3.5 [1ff4b]
p9a735 → ✓ @typescript-eslint/typescript-estree@npm:7.17.0 [30c28] doesn't provide @types/supports-color to debug@npm:4.3.5 [1ff4b]
pf7970 → ✓ @typescript-eslint/typescript-estree@npm:7.17.0 [30c28] doesn't provide supports-color to debug@npm:4.3.5 [1ff4b]
pe79cc → ✓ @typescript-eslint/typescript-estree@npm:7.17.0 [b3024] doesn't provide @types/supports-color to debug@npm:4.3.5 [1ff4b]
p51d27 → ✓ @typescript-eslint/typescript-estree@npm:7.17.0 [b3024] doesn't provide supports-color to debug@npm:4.3.5 [1ff4b]
p29faa → ✓ @typescript-eslint/utils@npm:7.17.0 [61552] doesn't provide @types/typescript to @typescript-eslint/typescript-estree@npm:7.17.0 [b3024] and 1 other dependency
pb4fbf → ✘ @typescript-eslint/utils@npm:7.17.0 [61552] doesn't provide typescript to @typescript-eslint/typescript-estree@npm:7.17.0 [b3024] and 1 other dependency
p5a8e9 → ✓ agent-base@npm:7.1.1 doesn't provide @types/supports-color to debug@npm:4.3.5 [1ff4b]
pcb7c4 → ✓ agent-base@npm:7.1.1 doesn't provide supports-color to debug@npm:4.3.5 [1ff4b]

`

pb4fbf → ✘ @typescript-eslint/utils@npm:7.17.0 [61552] doesn't provide typescript to @typescript-eslint/typescript-estree@npm:7.17.0 [b3024] and 1 other dependency

this make me hard ㅠ 

my package
   "@nestjs/cli": "^10.0.0",
    "@nestjs/schematics": "^10.0.0",
    "@nestjs/testing": "^10.0.0",
    "@types/express": "^4.17.17",
    "@types/jest": "^29.5.2",
    "@types/node": "^20.3.1",
    "@types/supertest": "^6.0.0",
    "@typescript-eslint/eslint-plugin": "^7.0.0",
    "@typescript-eslint/parser": "^7.0.0",
    "eslint": "^8.42.0",
    "eslint-config-prettier": "^9.0.0",
    "eslint-plugin-prettier": "^5.0.0",
    "jest": "^29.5.0",
    "prettier": "^3.0.0",
    "source-map-support": "^0.5.21",
    "supertest": "^7.0.0",
    "ts-jest": "^29.1.0",
    "ts-loader": "^9.4.3",
    "ts-node": "^10.9.1",
    "tsconfig-paths": "^4.2.0",
    "typescript": "^5.1.3"

**Comment by bradzacher** [comment]
As per our contributing guide we do not use discussions for support
https://typescript-eslint.io/contributing/discussions

Please review our contributing guide for ways to ask for support.
